src/data/faq.json

```diff
@@ -2696,7 +2697,7 @@
                                 ]
                             },
                             {
-                                "title": "I received the message that I had a risk encounter. The next day it was gone again. Shouldn't this be deleted after 14 days?",
+                                "title": "I received the message that I had a risk encounter. The next day it was gone again. Shouldn't this be deleted after 14 or resp. 10 days?",

```

Comment:
remove "resp." which is a German use of the word.

In English "respectively" is used to link two lists and it is not abbreviated like in German. For example you could say Version 2.19 and Version 2.20 use 14 and 10 days respectively. See https://dictionary.cambridge.org/dictionary/english/respectively for explanation.

Reply:
@amouzayanni 
I would recommend simply: "Shouldn't this be deleted after 14 or 10 days?"

Adding the version information in the title makes it more complicated. The text below explains about the version differences.